databuilder/tests/unit/models/feature/test_feature_generation_code.py

```diff
@@ -0,0 +1,60 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+
+from databuilder.models.feature.feature_generation_code import FeatureGenerationCode
+from databuilder.serializers import neo4_serializer
+
+
+class TestFeatureGenerationCode(unittest.TestCase):
```

Comment:
Same little comment explanation of why this is not just Query like for dashboards